docs/source/en/installation.md

```diff
@@ -50,6 +50,14 @@ pip install diffusers["flax"] transformers
 </jax>
 </frameworkcontent>
 
+## Install with conda
+
+After activating your virtual environment, with `conda` (maintained by the community):
```

Comment:
```suggestion
After activating your virtual environment, you can also install with `conda`:
```

Reply:
Now, is conda maintained officially rather than by the community?

Reply:
Good question, I don't think we officially maintain this but @patrickvonplaten or @sayakpaul would know more. 

If it is community maintained, then no need to make any changes :)